js/data/abstract_store.d.ts

```diff
@@ -40,11 +40,12 @@ export interface StoreOptions<T = Store> {
     /**
      * @docid
      * @type_function_param1 result:Array<any>
+     * @type_function_param2 loadOptions:LoadOptions
      * @action
      * @prevFileNamespace DevExpress.data
      * @public
      */
-    onLoaded?: ((result: Array<any>) => void);
+    onLoaded?: ((result: Array<any>, loadOptions: LoadOptions) => void);
```

Comment:
Is not `LoadOptions` an optional parameter?

Reply:
nope. See https://github.com/DevExpress/DevExtreme/blob/21_1/js/data/abstract_store.js#L83
even if options are not defined, the `loadOptions` initialized to an empty object before firing the callback